bin/instructions_makepdf.md

```diff
@@ -0,0 +1,98 @@
+
+### For Both Types You Have To
+
+- Provide a book template .adoc file under the `books` directory. Adopt to the requirements.
+- Copy the `build`, `fonts` and `resources` directory
+
+### Single Manual
+
+Adopt in `makepdf` following variables according the repo:
+```
+BOOK_TEMPLATE_NAME="ownCloud_Desktop_Client_Manual"
+BUILD_BASE_DIR="build/desktop"
+SPEAKING_NAME="Desktop Client"
+AVAILABLE_MANUAL="ROOT"
+```
+
+### Multi Manual
+
+Adopt in `makepdf` following variables according the repo:
+```
+BUILD_BASE_DIR="build/server"
+AVAILABLE_MANUALS=(admin user developer)
+```
+Be aware, that `AVAILABLE_MANUALS` is a bash array where the module names are separated by a blank!
+Consider that a module name is the name of an array element plus a trailing `_manual` string. This leads to a full name like `admin_manual` which must be a physical present directory located in `modules` directory containing the manual data.  
+
+### Debugging
+
+You will find in `makedf` the creation of the `param` variable. Some of those lines and others are commented. You can uncomment them based on your debugging requirements to print them to the console to see which options are created. This is useful if you test new params or if you run into build errors, see below.
+
+## Build Errors
+
+In very rare cases, it can happen that a pdf build errors. This should not be neglected and resolved. One issue identified (and solved) was a case when using a multi line source bash code block which trailed with a `\`. This  code block would not work in bash, should render properly but did not due to a backend bug, see [pdf creation fails when using a codeblock that ends with \ but has no following line with content](https://github.com/asciidoctor/asciidoctor-pdf/issues/1930). In such cases, use the debug mode `-d`, take the file and create a manual build on the command line adding all attributes and comment out all includes. Step by step reenable them and restart the build to see which document is failing. When the problematic file is found, make a copy of the file for backup purposes. Delete all content up to the first section and restart building. Copy back section by section and always do a build. When the problematic section is found do the same thing with the section content. When the issue has been identified, fix it to see if it will build successful. If a scheme is identifyable, fix them all and provide those fixes in the backup file. Finally you can drop the file and rename the backup file to its original name. A normal build should now succeed.
```

Comment:
```suggestion
In very rare cases, it can happen that a pdf build produces errors. This should not be neglected but resolved. One issue identified (and solved) was a case when using a multi-line source bash code block which trailed with a `\`. This  code block would not work in bash, should render properly but did not due to a backend bug, see [pdf creation fails when using a codeblock that ends with \ but has no following line with content](https://github.com/asciidoctor/asciidoctor-pdf/issues/1930). In such cases, use the debug mode `-d`, take the file and create a manual build on the command line, adding all attributes and commenting out all includes. Step by step re-enable them and restart the build to see which document is failing. When the problematic file is found, make a copy of the file for backup purposes. Delete all content up to the first section and restart building. Copy back section by section and always do a build. When the problematic section is found do the same thing with the section content. When the issue has been identified, fix it to see if it will build successfully. If a pattern is identifiable, fix all instances and provide those fixes in the backup file. Finally you can drop the debug file and rename the backup file to its original name. A normal build should now succeed.
```

Reply:
Oops, sorry the changes aren't showing up. Maybe too large a block?